app-info/eos-extra.xml

```diff
@@ -44,7 +44,7 @@
     <kudos>
       <kudo>GnomeSoftware::popular</kudo>
     </kudos>
-    <bundle type="flatpak"></bundle>
+    <bundle type="flatpak">app/com.google.Chrome//eos3</bundle>
   </component>
   <component type="desktop" merge="append">
     <id>com.microsoft.Skype.desktop</id>
```

Comment:
This is no more, we use Flathub's `com.skype.Client` now.

Reply:
Ah thanks!

Reply:
Should be addressed by https://github.com/endlessm/gnome-software-data/pull/14